docs/components/beacon/index.html

```diff
@@ -100,4 +99,67 @@ <h2 id="beacon-with-inline-styles">Beacon with Inline Styles</h2>
       </footer>
     </div>
   </section>
+
+  <section>
+    <header>
+      <h2 id="beacon-with-default-styles">Beacon with Color Override</h2>
+    </header>
+    <p>
+      This is an example implementation of <code>&lt;hx-beacon&gt;</code>
+      With color override.
+      Hexidecimal or named color values are valid, though hex values are
+      best practice for consistency.
+    </p>
+    <p class="hxSubdued hxSubBody">
+      <hx-icon type="exclamation-triangle"></hx-icon>
+        User can use <b>--hxBeacon</b> property for overriding the default Beacon
+        component color.
+        (Example:<b>--hxBeacon:#d32f2f or --hxBeacon:red</b>)
+    </p>
+    <div class="example" id="vue-beacon-default-styles" v-cloak>
+      <header>
+        <form class="beta-hxForm">
+          <fieldset>
+            <hx-text-control>
+              <input
+                id="txtBeaconDefaultStyles"
+                type="text"
+                v-model="beaconDefaultStyles"
+              />
+              <label for="txtBeaconDefaultStyles">
```

Comment:
Please change to reflect the demo - `txtBeaconOverrideStyles`.

Reply:
Changes have been done, Please review